demos/app/iot_demo_arguments.c

```diff
@@ -30,6 +30,7 @@
 /* Standard includes. */
 #include <stdlib.h>
 #include <string.h>
+#include <limits.h>
```

Comment:
Is this include needed now?  Maybe stdint.h should be used instead, I'm not sure how portable limits.h is.